wmidump.c

```diff
@@ -92,33 +92,35 @@ static int wmi_gtoa(const char *in, char *out)
 /*
```

Comment:
What is it you're trying to accomplish here? The new code has both longer lines and more of them. How is this simpler? Does it impact a future change? This isn't a refactor. Please describe your commits such that the first line is a summary of WHAT, the body expands on the WHAT and provides the WHY. If this is a dependency for a future patch, please make that clear here.